mlflow/spark.py

```diff
@@ -208,33 +215,26 @@ def log_model(
         spark_model = PipelineModel([spark_model])
     run_id = mlflow.tracking.fluent._get_or_start_run().info.run_id
     run_root_artifact_uri = mlflow.get_artifact_uri()
+    if _should_use_mlflowdbfs(run_root_artifact_uri):
+        mlflowdbfs_path = _mlflowdbfs_path(run_id, artifact_path)
+        with databricks_utils.MlflowCredentialContext(
+            get_databricks_profile_uri_from_artifact_uri(run_root_artifact_uri)
+        ):
+            try:
+                spark_model.save(mlflowdbfs_path)
+                return
+            except Py4JError as e:
```

Comment:
Are there any other types of exceptions that may be thrown if `mlflowdbfs` logging fails? Wonder if it's worth try / catching `Exception`.